src/FsCodec.SystemTextJson/Options.fs

```diff
@@ -59,12 +59,14 @@ type Options private () =
             /// <summary>Apply <c>UnionConverter</c> for all Discriminated Unions, if <c>TypeSafeEnumConverter</c> not possible. Defaults to <c>false</c>.</summary>
             [<Optional; DefaultParameterValue(null)>] ?autoUnionToJsonObject : bool) =
 
+        let defaultConverters: JsonConverter array  = [| StringConverter() |]
```

Comment:
there needs to be a way to opt out. See top level comment re considerations for defaulting to having it on, forcing a decision, or defaulting to off.
Whatever is done in the API here also needs to happen on the STJ side (unless it defaults to off, in which case it can be a new option that's just not been ported _yet_)

Reply:
I've made it optional for now

Reply:
I guess that's all good then!